src/Sylius/Bundle/AdminBundle/templates/product_review/form/sections/info/author.html.twig

```diff
@@ -0,0 +1,25 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon %}
+
+{% set author = hookable_metadata.context.resource.author %}
+
+<div class="card mb-3">
+    <div class="card-header">
+        <div class="card-title">
+            {{ 'sylius.ui.author'|trans }}
+        </div>
+    </div>
+    <div class="card-body">
+        <div class="card-title" {{ sylius_test_html_attribute('author-name') }}>
+            <a href="{{ path('sylius_admin_customer_update', {'id': author.id}) }}">{{ author.fullName }}</a>
+        </div>
+        <div class="card-subtitle">
+            {{ 'sylius.ui.customer_since'|trans }} {{ author.createdAt|date('Y-m-d') }}
+        </div>
+    </div>
+    <div class="card-footer">
+        <div class="py-1">{{ icon({icon: 'mail'}) }}<a href="mailto:{{ author.email }}">{{ author.email }}</a></div>
+        {% if author.phoneNumber is not empty %}
+            <div class="py-1">{{ icon({icon: 'phone'}) }} {{ author.phoneNumber }}</div>
+        {% endif %}
+    </div>
+</div>
```

Comment:
I'm wondering whether we should split this and the product elements into hooks, but it seems like a bit of an overkill atm. Thoughts?

Reply:
I wonder where the limit is, but I think splitting it up makes sense, so I opened the PR: #16332